### ISSUE TITLE
[Fix] vaccination data (1st, 2nd) was removed mistakenly

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#996]
## Summary
Fix the error in subsetting of vaccination data in `DataLoader.vaccine()`.

## Codes

```Python
cs.line_plot(vaccine_data.subset(country="Japan").set_index("Date"))
```

https://gist.github.com/lisphilar/bd342337784d1db37935296c756ad293

## Environment

- CovsirPhy version: 2.23.0
- Python version: 3.7.12
- Installation: pip
- System: Google Colaboratory

**Comment by lisphilar** [comment]
This should be fixed with #994 before updating to version 2.23.0, but not fixed.

**Comment by lisphilar** [comment]
This was fixed at version 2.23.0-alpha. 2.23.1 will be released.
![vaccinations](https://user-images.githubusercontent.com/7270139/153637455-5d798d09-fbf8-451b-8aa4-276676c79cfd.jpg)

**Comment by lisphilar** [comment]
Add variables regarding vaccinations:
```Python
snl.records(variables=["Vaccinations", "Vaccinated_once", "Vaccinated_full", "Vaccinations_boosters"])
```
![all](https://user-images.githubusercontent.com/7270139/153638041-5d6a65c7-58fa-4837-a6e8-2abe146e16a7.png)
